website/docs/cdktf/concepts/resources.mdx

```diff
@@ -122,6 +123,43 @@ class ResourceStack(TerraformStack):
             });
 ```
 
+```go
+import (
+	"fmt"
+
+	"github.com/aws/constructs-go/constructs/v10"
+	"github.com/aws/jsii-runtime-go"
+	"github.com/hashicorp/terraform-cdk-go/cdktf"
+	"github.com/hashicorp/terraform-cdk/examples/go/documentation/generated/hashicorp/aws/dataawsregion"
+	"github.com/hashicorp/terraform-cdk/examples/go/documentation/generated/hashicorp/aws/dynamodbtable"
+	aws "github.com/hashicorp/terraform-cdk/examples/go/documentation/generated/hashicorp/aws/provider"
+
+)
+
+func NewResourcesStack(scope constructs.Construct, name string) cdktf.TerraformStack {
+	stack := cdktf.NewTerraformStack(scope, &name)
+
+	aws.NewAwsProvider(stack, jsii.String("aws"), &aws.AwsProviderConfig{
+		Region: jsii.String("eu-central-1"),
+	})
+
+	region := dataawsregion.NewDataAwsRegion(stack, jsii.String("region"), &dataawsregion.DataAwsRegionConfig{})
+
+	dynamodbtable.NewDynamodbTable(stack, jsii.String("first-table"), &dynamodbtable.DynamodbTableConfig{
+		Name:    jsii.String(fmt.Sprintf("my-first-table-%s", *region.Name())),
```

Comment:
Not for this PR, but it's a bit of an oversight from JSII to not have the reverse`fromString`/`toString` functions 😅.

Reply:
What would those functions do?

Reply:
Since `jsii.String("foo")` results in a pointer to the string, `jsii.ToString` would convert a string pointer to a string, so:
`jsii.ToString(jsii.String("foo"))` would return `"foo"`

Reply:
Ah, got it. That requires no helper as in Go, you can do `*var` to resolve the pointer (see `*region.Name()` which does this as `Sprintf` wants a string and not a pointer for `%s`)

Reply:
Hmm, but that is true for `jsii.String` as well though, no? It's just `&"foo"` ([ref](https://github.com/aws/jsii-runtime-go/blob/main/helpers.go#L30))

Reply:
So, my comment was purely for aesthetic reasons :)

Reply:
> Hmm, but that is true for `jsii.String` as well though, no? It's just `&"foo"` ([ref](https://github.com/aws/jsii-runtime-go/blob/main/helpers.go#L30))

Sadly, it isn't 😅

<img width="702" alt="image" src="https://user-images.githubusercontent.com/1112056/216311517-353206c0-5a36-4339-8b93-487a2c60ea68.png">

Reply:
Using `&` only works on variables, that's why this utility exists :)

Reply:
Ah, gotcha. Makes sense. It saves you from creating an extra variable I suppose, but would still like there to be a reverse function without knowing the details of what `jsii.String` does. 

AWS SDK for Go does it too: https://docs.aws.amazon.com/sdk-for-go/api/aws/ (Value and Pointer Conversion Utilities)

Reply:
Ah, interesting! I bet you could discuss this with @RomainMuller 😄